crates/sui-indexer-alt-graphql/src/api/types/execution_error.rs

```diff
@@ -49,6 +49,15 @@ impl ExecutionError {
 
         Ok(Some(clever_error.source_line_number as u64))
     }
+
+    /// The instruction offset in the Move bytecode where the abort occurred. Populated for all Move aborts.
+    async fn instruction_offset(&self) -> Result<Option<u64>, RpcError> {
+        let ExecutionFailureStatus::MoveAbort(location, _) = &self.native else {
```

Comment:
`MovePrimitiveRuntimeError` may have location information too, which we can also support here.